src/shared/organisms/DataPanel/DataPanel.tsx

```diff
@@ -460,8 +460,12 @@ const DataPanel: React.FC<Props> = ({}) => {
             Boolean(resource.distribution?.contentSize)
               ? 'File'
               : 'Resource';
-          const contentType =
-            resource.distribution?.label?.split('.')?.pop() ?? '';
+          const contentType = getCorrectFileExtension(
+            resource.distribution?.label ?? '',
```

Comment:
it cannot be the case that `resource.distribution.label` is undefined (because we tackle take care of that in `toLocalStorageResources` function). So we can remove `?? ''`

Reply:
but it's possible that we can not have `distribution`  then resource.distribution will be `undefined`

Reply:
Well, actually, today it's not possible. The `toLocalStorageResources()` function normalizes the data such that there is always `distribution`

Reply:
make since, so the distribution object type must also change. 
I change the distribution type and the content type function